docs/en/tutorials/semi_det.md

```diff
@@ -0,0 +1,284 @@
+# Tutorial 14: Semi-supervised Object Detection
+
+The process of semi-supervised object detection is as follows:
+
+- Data Preparation
```

Comment:
Use the style of 'do something' to improve readability and use markdown syntax, we could also add reference to the titles, e.g.,
```
- [Prepare data splits](#Data Preparation)
```